components/search/ResultsList.js

```diff
@@ -306,7 +306,13 @@ const ResultItem = ({msmt}) => {
                   <ASNBox asn={msmt.probe_asn} />
                 </Box>
                 <Box width={4/16} title={moment.utc(msmt.measurement_start_time).format('YYYY-MM-DD HH:mm [UTC]')}>
-                  {moment.utc(msmt.measurement_start_time).format('YYYY-MM-DD HH:mm')}
+                  {intl.formatDate(moment.utc(msmt.measurement_start_time).toDate(), {
```

Comment:
You need to call the `useIntl` hook inside the FC to be able to use `intl` here.

Reply:
Sorry.  I misunderstood in the previous comment.